### PR TITLE
[12.0][ADD] web_group_by_percentage

### PR DESCRIPTION
**Pull request by tarteo**
This is a simple module which renders the share of each group in percentage with respect to the total sum.

image

### CODE REVIEW

**Review by tbaden — APPROVED** [review]
code review: lgtm

File: web_group_by_percentage/README.rst [hunk 1]
```diff
@@ -0,0 +1,84 @@
+===================
+Group By Percentage
```
Comment (pedrobaeza) [inline web_group_by_percentage/README.rst]: This is a bit misleading. You can put as title: `Show percentages over total in groups`

File: web_group_by_percentage/README.rst [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/162/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module shows the percentage of the total sum in group by rows.
+
+.. image:: /web_group_by_percentage/static/description/screenshot.png
```
Comment (pedrobaeza) [inline web_group_by_percentage/README.rst]: Please use relative path and let README generator (latest version) to change it to an absolute one for proper populating in all platforms.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Only little details

**Comment by tarteo** [comment]
@pedrobaeza Thanks, done.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**